.gitignore

```diff
@@ -14,3 +14,4 @@
 # limitations under the License.
 
 target
+.bsp
```

Comment:
Should we add some more lines and sort all lines?

```
.bloop
.idea
.metals
.vscode
project/metals.sbt
target
```

Reply:
We generally try to avoid adding lines to gitignore for things that depend on a users environment (for example what IDE they use). This way we don't have to keep track of all the possible IDE's and were they put their temporary files.

We have a comment in the Daffodil .gitignore that expands on this a bit and reccommends how IDE/OS specific files should be ignored. 

https://github.com/apache/daffodil/blob/main/.gitignore#L16-L22